package.json

```diff
@@ -51,9 +51,7 @@
     "ejs": "^3.1.3",
     "enquirer": "^2.3.5",
     "eslint": "^7.2.0",
-    "eslint-config-seek": "^7.0.1",
-    "eslint-plugin-jest": "^23.13.2",
-    "eslint-plugin-tsdoc": "^0.2.5",
```

Comment:
In particular, ESLint's docs seem to indicate that it shouldn't be possible to transitively bundle plugins 😕: https://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config

Reply:
🤞

Reply:
Seems to work 🧙